input/ig.xml

```diff
@@ -221,23 +221,23 @@
 				<reference value="AllergyIntolerance/lactose"/>
 			</reference>
 			<name value="AllergyIntolerance - Lactose intolerance"/>
-			<description value="Lactose (food) intolerance&#xa;&#xa;Patient: Mia BANKS"/>
+			<description value="Shows an example for the *AU Core AllergyIntolerance* profile.&#xa;&#xa;Patient: Mia BANKS"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-allergyintolerance"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="AllergyIntolerance/noneknown"/>
 			</reference>
 			<name value="AllergyIntolerance - No known allergy"/>
-			<description value="No known allergy&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows an allergyintolerance example for the *AU Core AllergyIntolerance* indicating the patient has no known allergies.&#xa;&#xa;Patient: Mr. Li Wang"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-allergyintolerance"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="AllergyIntolerance/penicillin"/>
 			</reference>
 			<name value="AllergyIntolerance - Penicillin allergy"/>
-			<description value="Penicillin (medication) allergy&#xa;&#xa;Patient: Deangelo HOWE"/>
+			<description value="Shows an example for the *AU Core AllergyIntolerance* profile.&#xa;&#xa;Patient: Deangelo HOWE"/>
```

Comment:
This description is insufficient in terms of providing information on the content.